### PR TITLE
style(tests): mixed lint updates

### PR DESCRIPTION
**Pull request by layershifter**
This PR contains only styling changes, it was splitted from #1895 to simplify review.

### CODE REVIEW

File: test/specs/commonTests/isConformant.js [hunk 1]
```diff
@@ -264,10 +264,10 @@ export default (Component, options = {}) => {
 
         // Components should return the event first, then any data
         handlerSpy.calledWithMatch(...expectedArgs).should.equal(true,
-          `<${constructorName} ${listenerName}={${handlerName}} />\n` +
+          `${`<${constructorName} ${listenerName}={${handlerName}} />\n` +
           `${leftPad} ^ ${errorMessage}\n` +
-          'It was called with args:\n' +
-          JSON.stringify(handlerSpy.args, null, 2)
+          'It was called with args:\n'}${
+            JSON.stringify(handlerSpy.args, null, 2)}`,
```
Comment (levithomason) [inline test/specs/commonTests/isConformant.js]: Hm, looks like the update here went awry.

File: test/specs/lib/numberToWord-test.js [hunk 2]
```diff
@@ -26,7 +26,7 @@ describe('numberToWord', () => {
   })
 
   it('returns word if input is word', () => {
-    words.forEach((word) => numberToWord(word).should.equal(word))
+    words.forEach(word => numberToWord(word).should.equal(word))
```
Comment (levithomason) [inline test/specs/lib/numberToWord-test.js:29]: Why is it that in some (most?) cases the parens are added, but in others they are removed?

https://github.com/Semantic-Org/Semantic-UI-React/pull/1939/files#diff-ca99bb99c3b7ee63716bc6db755e5413R107

https://github.com/Semantic-Org/Semantic-UI-React/pull/1939/files#diff-ca99bb99c3b7ee63716bc6db755e5413R107

..etc

File: test/tests.bundle.js [hunk 3]
```diff
@@ -9,7 +9,7 @@ const testsContext = require.context('./', true, /-test\.js$/)
 // only re-run changed tests, or all if none changed
 // https://www.npmjs.com/package/karma-webpack-with-fast-source-maps
 const __karmaWebpackManifest__ = []
-let runnable = testsContext.keys().filter((path) => __karmaWebpackManifest__.indexOf(path) >= 0)
+let runnable = testsContext.keys().filter(path => __karmaWebpackManifest__.indexOf(path) >= 0)
```
Comment (levithomason) [inline test/tests.bundle.js:12]: Here's another example where parens are removed while most others are added.

**Review by levithomason — CHANGES_REQUESTED** [review]
I believe there is syntax error in `isConformant.js`, see inline comment.

Other than this, it seems the style is also inconsistent between files (arrow func arg parens).  That said, I'd rather implement `eslint` rules that fixable for these so we don't have to manage them ourselves.

File: test/specs/lib/numberToWord-test.js (hunk 2 above)
Comment (layershifter) [reply test/specs/lib/numberToWord-test.js:29]: https://github.com/airbnb/javascript#arrows--implicit-return

File: test/specs/commonTests/isConformant.js (hunk 1 above)
Comment (layershifter) [reply test/specs/commonTests/isConformant.js]: Yep, should be fixed. Will fix tomorrow

**Comment by layershifter** [comment]
@levithomason  this come from airbnb rules, I gave link in a comment:

> If the function body consists of a single statement returning an expression without side effects, omit the braces

**Comment by levithomason** [comment]
That would refer to the curly braces of the function body`{}`, I'm referring to the parentheses of the arguments `()`.

When there is a single argument, the changes here arbitrarily add parens in some cases and remove them in other cases.   I can't seem to figure out what the reason is.

**tests.bundle.js** (Removes parens)
```diff
-.filter((path) =>
+.filter(path =>
```

**test/specs/views/Feed/FeedEvent-test.js** (Adds parens)
```diff
-contentProps.forEach(propKey => {
+contentProps.forEach((propKey) => {
```

**test/specs/lib/numberToWord-test.js** (Removes parens)
```diff
-words.forEach((word) =>
+words.forEach(word =>
```

**test/specs/behaviors/Visibility/Visibility-test.js** (Adds parens)
```diff
-expectations.forEach(expectation => {
+expectations.forEach((expectation) => {
```

...there are many of these in the PR.

**Comment by layershifter** [comment]
@levithomason This code was automatically fortmatted with `eslint` and `airbnb` preset. I split #1895 to smaller PRs only for review purposes.

It looks confusing, but it's correct. Take a look [8.2](https://github.com/airbnb/javascript#arrows--implicit-return), this paragraph affects `arrow-parens` rule, too.

```js
// bad
[1, 2, 3].map(number => {
  const nextNumber = number + 1;
  `A string containing the ${nextNumber}.`;
});

// good
[1, 2, 3].map(number => `A string containing the ${number}.`);

// good
[1, 2, 3].map((number) => {
  const nextNumber = number + 1;
  return `A string containing the ${nextNumber}.`;
});

// good
[1, 2, 3].map((number, index) => ({
  [index]: number,
}));
```

Paragraph [8.4](https://github.com/airbnb/javascript#arrows--one-arg-parens) also describes behaviour of parentheses.

```js
// bad
[1, 2, 3].map((x) => x * x);

// good
[1, 2, 3].map(x => x * x);
```

File: test/specs/commonTests/isConformant.js (hunk 1 above)
Comment (layershifter) [reply test/specs/commonTests/isConformant.js]: I've fixed this.

**Review by layershifter — COMMENTED** [review]
@levithomason I've fixed formatting of that message and answered about parens.

**Comment by levithomason** [comment]
Got it, thank you again.  I'm OK with this so long as it is fully automated.  👍

**Comment by levithomason** [comment]
Released in `semantic-ui-react@0.71.5`

**Pull request merged**